src/js/features/license/selectors.js

```diff
@@ -5,4 +5,9 @@ export const error = state => state.license.error;
 export const txHash = state => state.license.txHash;
 export const licenseOwners = state => state.license.licenseOwners;
 export const licenseOwnersError = state => state.license.licenseOwnersError;
-export const getLicensePrice = state => parseInt(state.license.price, 10);
+export const getLicensePrice = state => {
+  if (state.license.price === Number.MAX_SAFE_INTEGER) {
+    return "price_error";
+  }
```

Comment:
This is supposed to return a number, so we probably have bad error handling in terms of when we get the prices.